app/modules/sightings/resources.py

```diff
@@ -513,3 +515,55 @@ def delete(self, sighting):
         # TODO handle failure of feather deletion (when edm successful!)  out-of-sync == bad
         sighting.delete_cascade()
         return None
+
+
+@api.route('/<uuid:sighting_guid>/featured_asset_guid')
```

Comment:
again, should this be camelcase or not? Not sure

Reply:
I am not sure. There are precious few examples like this:

Reply:
I am not sure. There are precious few examples of a route with a segment in snake case: 

`@api.route('/admin_user_initialized')`
and
`@api.route('/**src_raw**/<uuid:asset_guid>', doc=False)`

But I don't see _any_ with a segment in camelCase. I don't think it's a big deal as we (myself included of course) should avoid building a route this way in the future per recent conversation.